packages/devtools_app/lib/src/standalone_ui/vs_code/flutter_panel.dart

```diff
@@ -190,6 +192,8 @@ class _EditorConnectedPanelState extends State<_EditorConnectedPanel>
                   editor: widget.editor,
                   debugSessions: debugSessions,
                 ),
+              if (FeatureFlags.propertyEditor)
+                const PropertyEditorSidebar(),
```

Comment:
we should put the property editor into its own VS code sidebar and not the Flutter panel sidebar. So that would look like adding another standalone ui endpoint